mlflow/gateway/providers/openai.py

```diff
@@ -153,6 +156,11 @@ async def completions(self, payload: completions.RequestPayload) -> completions.
             raise HTTPException(
                 status_code=400, detail="Invalid parameter `n`. Use `candidate_count` instead."
             )
+
+        # Set default values if not set at call time
+        payload.setdefault("temperature", 0.0)
+        payload.setdefault("candidate_count", 1)
```

Comment:
Why do we need this? `temperature` doesn't default to `0.0`?

Reply:
I see

https://github.com/mlflow/mlflow/pull/9199/files#diff-d2379e060ce89b84cbfeb5db4e40737819407c22cb8c4e26b17cadaa37d9b82bR16-R17